### DISCUSSION TITLE
Suppress 'Could not find "window.__TAURI_METADATA__". ' log

### DISCUSSION THREAD

**Discussion by Daniel-Knights** [tauri-apps/tauri#4356] [Q&A]
I'm using Vitest and every time I run my suites there's a wall of these logs:

```bash
stderr | unknown test
Could not find "window.__TAURI_METADATA__". The "appWindow" value will reference the "main" window label.
Note that this is not an issue if running this frontend on a browser instead of a Tauri window.
```

Makes it really awkward to debug. Is there any way to suppress them?

**Comment by JonasKruckenberg** [comment]
This is caused by the `appWindow` constant, that evaluates code when you import the module (we want to deprecate the constant for v2).

The workaround currently is to call `mockWindows("main")` in a `beforeEach` hook. If you need to mock specific window labels you can use the function in tests again and it will override the label

**Reply by Daniel-Knights** [reply]
Hi Jonas, thanks for the response. I tried adding this to every test suite, but I'm still getting the same output:

<img width="1440" alt="Screenshot 2022-06-16 at 06 51 39" src="https://user-images.githubusercontent.com/59598622/174000853-645cb658-daa8-4552-8632-496a43012d22.png">

**Reply by Daniel-Knights** [reply]
This is all I have in `vite.config.ts`:

```ts
  test: {
    environment: 'jsdom',
    globals: true,
  },
```
